Add Audio permission

llewelld opened the pull request:
This change adds the "Audio" permission so that the "Media Player scanner" option works when the app is Sailjailed. This is a really nice feature, so personally I think it's worth adding the permission for.

For broader context: the permission allows the app to access the MPRIS metadata. From what I can tell, the permission is really intended for apps that want to show info on the homescreen (using MPRIS) but it seems to work for reading the MPRIS data too.

For reference:
1. [Audio permission description](https://github.com/sailfishos/sailjail-permissions#permissions)
2. [MPRIS dbus access](https://github.com/sailfishos/sailjail-permissions/blob/2cdb5e7f739e99ad4fd12e80cb1558eea4167f8e/permissions/Audio.permission#L29)

llewelld committed:
Add Audio permission

Reading the MPRIS data requires the Audio permission to be added. This
is used for the "Media Player scanner" option.

This change therefore switches the Music for the Audio permission.

ilpianista commented:
Oh, I thought [Music](13094fec) was enough to get MPRIS? If not, please remove it while you are in it. Thanks!

llewelld commented:
Thanks for taking a look at this. I removed the Music permission and have been using the app for a bit, and haven't notice any ill-effects, so I guess Music isn't needed. I've updated the PR.

ilpianista commented:
Thanks!
